packages/components/src/custom-select-control/index.js

```diff
@@ -103,6 +105,20 @@ export default function CustomSelectControl( props ) {
 
 	const [ isFocused, setIsFocused ] = useState( false );
 
+	function handleOnFocus( e ) {
+		setIsFocused( true );
+		if ( typeof onFocus === 'function' ) {
+			onFocus( e );
+		}
+	}
+
+	function handleOnBlur( e ) {
+		setIsFocused( false );
+		if ( typeof onBlur === 'function' ) {
+			onBlur( e );
+		}
```

Comment:
```suggestion
		onBlur?.( e );
```

Maybe we could use optional chaining here as well?

Reply:
done.